### ISSUE TITLE
a11y and audio

### ISSUE THREAD

**Issue by VeryGoodErotica** [w3c/a11y-discov-vocab#28]
At the moment I can't reach the actual web page that has it (browser can't find idpf.org) but archive.org had it - in 'META-004: Identify accessibility hazards' with respect to audio it mentions:

    sound — certain sound patterns, such as ringing and buzzing, can cause seizures.

There does not seem to be an expansion on that, for how to evaluate an audio as to whether or not it may have one of the patters that can be a trigger. I tried searching the web for an evaluation method but did not find one.

Can the doc be updated to give a reference for evaluation?

-=-

Also somewhat related, maybe belongs with WAI for inclusion in their WCAG general techniques and not here - many people (myself included) have an autistic input overload trigger caused by either loud audio or sudden changes in audio volume. I couldn't watch Big Bang Theory because they mixed the laugh track to be really loud compared to the rest of the show, for example, causing an input overload that stressed me.

Using loudness normalization as specified in EBU R128 should be an accessibility technique for audio and video - either by providing the metadata tags or (what I do with audio) applying EBU R128 to the source before lossy encoding, so that it is loudness normalized even if the player doesn't support the metadata tags.

EBU R128 isn't the only scheme but it is the best (ReplayGain for example I believe only takes RMS into consideration which is often inaccurate method) and EBU R128 seems to be gaining a lot of support in the broadcasting industry, including streaming services, so if a user has their volume set what they like when listening to spotify and they then open an ePub with an audio or video that uses EBU R128 the perceived loudness will be where they want it.

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/publishing/groups/epub-wg/Meetings/Minutes/2021-11-11-epub-a11y#section3) on 2021-11-11

List of resolutions:

- ***[Resolution No. 1](https://www.w3.org/publishing/groups/epub-wg/Meetings/Minutes/2021-11-11-epub-a11y#resolution1):  Approve PR 1903.***

<details><summary><i>View the transcript</i></summary>

### 3. a11y and audio: #1302.

**Matt Garrish:** almost a given that this will happen at this point.  

_See github issue [epub-specs#1302](https://github.com/w3c/a11y-discov-vocab/issues/28)._




**Avneesh Singh:** issue is about audio hazards, which are underspecified currently.  

_See github pull request [epub-specs#1903](https://github.com/w3c/epub-specs/pull/1903)._




**Avneesh Singh:** what should be considered an audio hazard?.  

**Matt Garrish:** we're referencing here the Schema metadata (which was formerly IMS metadata), so not much we can do.  
… lack of standard on when audio constitutes a hazard.  
… next step will be to pursue this in the new vocabulary CG.  

**Avneesh Singh:** if we want to put pressure on WAI, would it make more sense to keep this issue in epub 33 wg.  

**Matt Garrish:** we should maybe open an issue in WAI tracker.  

> *Tzviya Siegman:* +1 to mgarrish.

**Matt Garrish:** you sent an email to them earlier this year to put it on their radar, but it may have been forgotten now.  

> **Proposed resolution: Approve PR 1903.** *(Avneesh Singh)*

> *Matt Garrish:* +1.

> *Wendy Reid:* +1.

> *Matthew Chan:* +1.

> *Gregorio Pellegrino:* +1.

> *Wendy Reid:* george: +1.

> *Murata Makoto:* +1.

> ***Resolution #1: Approve PR 1903.***

**John Foliot:** i would recommend that we not take this to a11y guidelines wg, it'll get lost.  
… they are very busy now.  
… maybe take this to APA, research questions TF.  

**Matt Garrish:** great! Thank you..  

**John Foliot:** two categories of sound hazards I can think of: 1) auto-start/loud, and 2) specific types of sounds like explosions.  
… this sort of thing is perfect for the research questions TF.  

**Avneesh Singh:** APA is also more research oriented.  

**John Foliot:** AG is focused on a11y scoring issue right now. You can log the issue against WCAG3 if you want, but I would not recommend.  

**George Kerscher:** maybe this is something that can go into DaisyKB, that people are aware of volumes, that if there are sudden amplitude jumps that a hazard should be entered into epub metadata.  

**Matt Garrish:** yeah, we can certainly do that. Within our domain..  

**Avneesh Singh:** we will discuss that further via email.  
… for now mgarrish please open the issue with APA.  
… and we may point AG to this issue again in future.  
… and also email WAI group to raise this question.  

</details>
